README.md

```diff
@@ -39,6 +38,7 @@ The following images are currently published and updated by the distroless proje
 | gcr.io/distroless/python3-debian12    | latest, nonroot, debug, debug-nonroot | amd64, arm64                      |
 | gcr.io/distroless/java-base-debian12  | latest, nonroot, debug, debug-nonroot | amd64, arm64, s390x, ppc64le      |
 | gcr.io/distroless/java17-debian12     | latest, nonroot, debug, debug-nonroot | amd64, arm64, s390x, ppc64le      |
+| gcr.io/distroless/java21-debian12     | latest, nonroot, debug, debug-nonroot  | amd64, arm64, ppc64le            |
```

Comment:
@sparsick Just noticed... maybe you like to remove the excess blank. 😄

Reply:
Is this causing a failure on the lint check? I don't see those actions running.